presto-docs/src/main/sphinx/release/release-301.rst

```diff
@@ -0,0 +1,49 @@
+===========
+Release 301
+===========
+
+General Changes
+---------------
+
+* Fix reporting of aggregate input data size stats. (:issue:`100`)
+* Add support for role management (see :doc:`/sql/create-role`).  Note, using :doc:`/sql/set-role` requires an up-to-date client library. (:issue:`90`)
+* Add ``INVOKER`` security mode for :doc:`/sql/create-view`. (:issue:`30`)
+* Add ``ANALYZE`` SQL statement for collecting table statistics. (:issue:`99`)
+* Add :func:`log` function with arbitrary base. (:issue:`36`)
+* Remove the ``deprecated.legacy-log-function`` configuration option. The legacy behavior for the :func:`log` function is no longer available. (:issue:`36`)
+* Remove the ``deprecated.legacy-array-agg`` configuration option. The legacy behavior for :func:`array_agg` is no longer available. (:issue:`77`)
+* Disallow uninstalling Presto RPM while server is still running. (:issue:`67`)
+* Improve performance of `coalesce` expressions. (:issue:`35`)
```

Comment:
Double backticks and uppercase since it's a keyword (consistent with other usages)